keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -487,6 +488,24 @@ free to use non-namespaced properties (e.g. `fingerprint`) as they see fit, but
 any shared tooling should use appropriately namespaced names.
 
 
+#### Property Ladder
+
+We foresee that there will be more properties that the community wants to adopt as "well known" properties. Thus, we want to define a process for adopting a property to be a well known property in the future:
+
+1. **Extension Property Stage**: A property is implemented as an extension property if the implementer has no intent to elevate it to a standard property. This property cannot use the reserved suffixes and is not required to be implemented by all implementers.
+
+2. **Standard Property Stage**: The implementor can propose to add a property to be a **standard** property which means if anyone implements the property with the same name, it must follow the same definition and rules described in the KEP. Not all implementers are required to implement the property.
+   The process for proposing a standard property is as follows:
+    * The property sponsor must present a discussion agenda with clear use cases and motivation at a Multi-cluster SIG meeting.
+    * The community will vote on the proposal, and if it receives approval from at least 2/3 of voters, the property becomes a standard property and is incorporated into the About API KEP.
+
+3. **Core Property Stage**: After a property has maintained standard status for at least 3 months and has 3 or more implementations, one can propose to elevate it to a **core** property which means it must be implemented by every implementer that implements the About API.
```

Comment:
what happens if it is missing? 3months from? given we don't have a release process so it may be hard to start counting?

Should we have some kind of versioning in place?

If we don't have versioning, we may want more than 3mo of stability.